job_server/src/test/mailingList.spec.ts

```diff
@@ -0,0 +1,65 @@
+import app from "../server";
+import * as chai from "chai";
+import chaiHttp = require("chai-http");
+import "mocha";
+import {createConnection} from "typeorm";
+
+chai.use(chaiHttp);
+const expect = chai.expect;
+const testEmail: string = "test01@gmail.com";
+
+describe("Mailing list API tests", () => {
+    before((done) => {
```

Comment:
What does done() do? I haven't seen it used in Mocha/Chai tests before and these tests worked when I took it out too.

Reply:
`done()` is for tests that have async code. I think it is necessary because the tests use promises, so the test would think it's done after it's created the promise, not after all the promise code has run